react-app/package.json

```diff
@@ -11,18 +14,23 @@
     "@testing-library/user-event": "^7.2.1",
     "@types/jest": "^24.9.1",
     "@types/lodash": "^4.14.157",
+    "@types/material-ui": "^0.21.7",
     "@types/node": "^12.12.47",
     "@types/react": "^16.9.41",
     "@types/react-dom": "^16.9.8",
     "@types/react-router": "^5.1.8",
     "@types/react-router-dom": "^5.1.5",
     "@types/react-transition-group": "^4.4.0",
     "clsx": "^1.1.1",
+    "date-fns": "^2.15.0",
     "lodash": "^4.17.19",
+    "nivo": "^0.31.0",
     "postcss-cli": "^7.1.1",
     "react": "^16.13.1",
+    "react-calendar": "^3.1.0",
```

Comment:
Is this needed after now Material UI is the date picker?

Reply:
Same for row 33

Reply:
Yeah that's a good one.  I will remove them.

Reply:
Thanks :)